examples/tutorials/shader-modules/app.ts

```diff
@@ -3,6 +3,7 @@
 // Copyright (c) vis.gl contributors
 
 import {Buffer, NumberArray} from '@luma.gl/core';
+// @ts-expect-error
```

Comment:
Is this temporary? Add TODO?

Reply:
Yes, like much of the other issues you've picked up. PR is still WIP :)